Plugins/GitFlow/GitFlowForm.cs

```diff
@@ -160,7 +161,7 @@ private void DisplayBranchDatas()
         private void LoadBaseBranches()
         {
             var branchType = cbType.SelectedValue.ToString();
-            var manageBaseBranch = (branchType == Branch.feature.ToString("G") || branchType == Branch.hotfix.ToString("G") || branchType == Branch.support.ToString("G"));
+            var manageBaseBranch = (branchType == Branch.feature.ToString("G") || branchType == Branch.hotfix.ToString("G") || branchType == Branch.support.ToString("G") || branchType == Branch.bugfix.ToString("G"));
```

Comment:
if `manageBaseBranch` is anything but `Branch.release` shouldn't it be simplified to

```
var manageBaseBranch = branchType != Branch.release.ToString("G");
```